code/game/objects/items/devices/flash.dm

```diff
@@ -78,15 +78,15 @@
 
 		var/safety = M:eyecheck(TRUE)
 		if(safety <= 0)
-			M.Weaken(10)
 			flick("e_flash", M.flash)
 				//Vaurca damage 15/01/16
 			var/mob/living/carbon/human/H = M
 			if(isvaurca(H))
 				var/obj/item/organ/eyes/E = H.get_eyes()
 				if(!E)
 					return
-				usr << span("alert", "Your eyes burn with the intense light of the flash!.")
+				usr << span("alert", "Your eyes burn with the intense light of the flash!")

```

Comment:
`user` not `usr`. Ensure there are no other instances of `usr` where `user` should be used.

Reply:
was there before, i didn't write it.